CompilerSource/compiler/components/write_font_info.cpp

```diff
@@ -41,7 +41,11 @@ using namespace std;
 int lang_CPP::compile_writeFontInfo(const GameData &game)
 {
   ofstream wto((codegen_directory + "Preprocessor_Environment_Editable/IDE_EDIT_fontinfo.h").c_str(),ios_base::out);
-  wto << license << "#include \"Universal_System/Resources/fonts_internal.h\"" << endl
+  wto << license
```

Comment:
This file is never included during a JDI run. You don't need to edit the compiler at all. You can just `#undef INCLUDED_FROM_SHELLMAIN` at the top of the resource file include block in `SHELLmain`.